.github/workflows/release.yml

```diff
@@ -32,13 +32,52 @@ on:
           - minor
           - major
 jobs:
-  release:
+  rc:
+    if: ${{ inputs.type == 'rc' }}
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@master
       - name: Prepare package
         run: |
           git pull
+          git checkout ${{ inputs.rc_branch_name }}
+          git config --global user.name "${GITHUB_ACTOR}"
+          git config --global user.email "${GITHUB_ACTOR_ID}+${GITHUB_ACTOR}@users.noreply.github.com"
           yarn
           yarn build
           yarn test
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "20.x"
+          registry-url: "https://registry.npmjs.org"
+          always-auth: true
+      - name: Publish to npm
+        run: yarn publish --tag ${{ inputs.rc_version_tag }} --new-version ${{ inputs.rc_version_name }}
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+  latest:
+    if: ${{ inputs.type == 'latest' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Prepare package
+        run: |
```

Comment:
Visto che i due comandi `run` si differenziano per un solo comando, non so se sia possibile creare una sorta di comando unico da richiamare in tutti e due i punti, per evitare la duplicazione del codice.
Magari da vedere anche nella prossima storia in cui si andrà a lavorare sulla release note.